### PR TITLE
Profile grid

### LINKED ISSUE #48: Refactor code, clean up - Create Page
# Value Statement
As a user seeing the create page for the first time, I want to create a flowing form that is responsive and easy to read so that the user can enjoy the experience of filling out their profile details.

# Description

Screenshot 2021-05-12 at 11 16 25

# Acceptance Criteria

- The form should be styled.
- The code for this page should be tidied with components and imported functions used where possible.

# Tasks
- [x] Create a better structured form with `display: grid;`.
- [x] Move large chunks of code to components.
- [x] Where applicable, import functions.
- [x] Refactor where possible.

# Complexity
Medium/Low

### PR DESCRIPTION
**Pull request by hugh-burgess**
- Grid implemented on MainContent.js
- Profile, Edit and Tour Page corresponding to the Grid.

### CODE REVIEW

File: client/src/components/profile-page/Profile.css [hunk 1]
```diff
@@ -4,111 +4,139 @@
   background-color: var(--light-yellow);
 }
 
-.profile-wrapper {
+.grid-wrapper {
+  margin-top: 2em;
+  max-height: 100vh;
+  max-width: 100vw;
```
Comment (yvlange) [inline client/src/components/profile-page/Profile.css]: the max-width doesn't seem to work 100%. On the mobile view the content doesn't stay in the viewport:

Screenshot 2021-05-14 at 07 59 58

Maybe `overflow: hidden`fixes it?!

File: client/src/components/profile-page/Profile.css [hunk 2]
```diff
@@ -4,111 +4,139 @@
...
-  top: 90px;
-  z-index: 1;
-  font-size: 36px;
+  font-size: 2.5rem;
   color: var(--deep-blue);
 }
 
+.profile-bubble {
+  height: 130%;
+  margin-left: -14rem;
```
Comment (yvlange) [inline client/src/components/profile-page/Profile.css:69]: as mentioned in the previous comment, the bubble doesn't stay in the viewport

File: client/src/components/profile-page/Profile.css [hunk 3]
```diff
@@ -4,111 +4,139 @@
...
-  position: relative;
+  flex-direction: column;
+  width: 100%;
+  text-align: center;
+  align-self: flex-end;
+  margin-bottom: -2em;
+  z-index: 10;
+}
+
+.feedback {
```
Comment (yvlange) [inline client/src/components/profile-page/Profile.css:123]: I would add a `margin` here, so the headline isn't so close to the bubble _(only stylistic advice haha)_

**Review by yvlange — APPROVED** [review]
Good job @hugh-burgess ! Only left some small comments :)

**Review by FriederikeM — APPROVED** [review]
Something is off in the profile page. Things are not in their bubbles 
Screenshot 2021-05-14 at 10 15 00

**Pull request merged**